sys/bloom/bloom.c

```diff
@@ -35,7 +35,7 @@ bloom_t *bloom_new(size_t size, size_t num_hashes, ...)
     }
 
     /* Allocate Bloom array */
-    if (!(bloom->a = calloc(ROUND(size), sizeof(char)))) {
+    if (!(bloom->a = calloc(sizeof(uint8_t), ROUND(size)))) {
```

Comment:
@Kijewski, @gebart, @cgundogan, @authmillenon, we need a consensus on this debate to merge this PR until tonight, otherwise we cannot include in the release candidate and this fix won't be part of the release.